app/(website)/layout.tsx

```diff
@@ -17,7 +17,7 @@ export default async function WebsiteLayout({ children }: Props) {
       {/* <pre>{JSON.stringify(navbarLinks, null, 2)}</pre> */}
       <main>{children}</main>
       {/* TODO: Agregar componente Footer */}
-      {/* <pre>{JSON.stringify(footerLinks, null, 2)}</pre> */}
+      <pre>{JSON.stringify(footerLinks, null, 2)}</pre>
```

Comment:
Hello @alexxuss1996!. You need to replace these two lines with the `Footer` component.

Also... Please, try to ask to be assigned to issues you want to work on (beforehand), that way others know. We will probably add a lot more issues starting this week now that Nykka just returned from vacation.

I'll be reviewing the rest of Pull Requests tomorrow (monday) :rocket:

Reply:
Oh, I forgot to save the last change in `layout.tsx`...
Also, sorry for the inconvenience!